### DISCUSSION TITLE
how to install latest version via yum

### DISCUSSION THREAD

**Discussion by clearcodecn** [containers/podman#17843] [General]
### Issue Description

while I install podman by `yum install podman`, 
it install `podman version 1.6.4`

how can I install v4.0 and later on centos.

### Steps to reproduce the issue

Steps to reproduce the issue
1.
2.
3.

### Describe the results you received

Describe the results you received

### Describe the results you expected

Describe the results you expected

### podman info output

```yaml
If you are unable to run podman info for any reason, please provide the podman version, operating system and its version and the architecture you are running.
```

### Podman in a container

No

### Privileged Or Rootless

None

### Upstream Latest Release

Yes

### Additional environment details

Additional environment details

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

**Comment by saril009** [comment]
it looks like this is the latest version of podman available for rhel7 
podman version 1.6.4 
is there any other yumrepo for same ?

**Reply by baude** [reply]
it will not be a simple exercise to just put a newer podman on centos7.  you will need a handful of utilities to be added which are also not present or new enough.
